### ISSUE TITLE
Dropout implementation in recurrent-visual-attention

### ISSUE THREAD

**Issue by jrbtaylor** [Element-Research/rnn#156]
In examples/recurrent-visual-attention there is a command-line option for dropout in the hidden layer but it doesn't appear to be implemented anywhere in the code. Is one of the dp modules handling this or is it not implemented yet? Would it not simply be putting `recurrent `in a `nn.Sequential()` container on line 125 and then adding `recurrent:add(nn.Dropout(0.5))` to line 129?

**Comment by nicholas-leonard** [comment]
That cmd-line argument shouldn't be there. I just removed it in latest commit. REINFORCE makes the model stochastic enough, dropout would make it too stochastic I think.
